astropy/table/tests/test_operations.py

```diff
@@ -1829,13 +1848,13 @@ def test_masking_required_exception():
     Test that outer join, hstack and vstack fail for a mixin column which
     does not support masking.
     """
-    col = [1, 2, 3, 4] * u.m
+    col = table.NdarrayMixin([0, 1, 2, 3])
     t1 = table.QTable([[1, 2, 3, 4], col], names=['a', 'b'])
     t2 = table.QTable([[1, 2], col[:2]], names=['a', 'c'])
 
     with pytest.raises(NotImplementedError) as err:
         table.vstack([t1, t2], join_type='outer')
-    assert 'vstack requires masking' in str(err.value)
+    assert 'vstack unavailable' in str(err.value)
```

Comment:
Change this to `pytest.raises(NotImplementedError, match='vstack unavailable')`?

Reply:
I noticed this old code, but was trying to keep the footprint small and not make changes just for style.